fury-on-gke/README.md

```diff
@@ -521,7 +599,7 @@ Navigate to <http://directory.fury.info> to see all the other ingresses deployed
 
 [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards) is an open-source analytics and visualization platform for OpenSearch. OpenSearch Dashboards lets you perform advanced data analysis and visualize data in various charts, tables, and maps. You can use it to search, view, and interact with data stored in OpenSearch indices.
 
-Navigate to <http://logs.fury.info> or click the OpenSearch Dashboards icon from Forecastle.
+Navigate to <http://opensearch-dashboards.fury.info> or click the OpenSearch Dashboards icon from Forecastle.
 
 #### Read the logs
```

Comment:
I've improved this section in the other guides, I'd recommend porting the changes over to this guide also. See [#27](https://github.com/sighupio/fury-getting-started/issues/27)